docs/smart-contracts/marlowe.md

```diff
@@ -29,6 +29,63 @@ The design guarantees the following:
 The Marlowe Playground is a plug-and-play smart contract builder and simulator that is simple to use, visual, and modular. Build, simulate, and analyze Marlowe contracts in this 4-minute tour of the Marlowe Playground.
 <iframe width="100%" height="325" src="https://www.youtube.com/embed/axP-jYQ_6lo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture fullscreen"></iframe>
 
+## Resources for Developing and Deploying Marlowe Contracts
+- How do I run my Marlowe contract on the Cardano blockchain?
+	1. Design your contract using [Marlowe Playground](https://play.marlowe-finance.io/#/).
+	2. Press the `Send to Simulator` button and then press `Download as JSON` to download your contract in JSON format.
+	3. *If you want to run your contract at the command line using `marlowe-cli`,* install [`marlowe-cli`](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-cli/ReadMe.md#installation) and follow the instructions [in the tutorial for running contracts with `marlowe-cli`](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-cli/lectures/04-marlowe-cli-concrete.md). [Here](https://www.youtube.com/playlist?list=PLNEK_Ejlx3x0GbvCw-61e9VfRafBT1JSw) is a video lecture that provides an overview of `marlowe-cli`.
```

Comment:
```suggestion
	3. *If you want to run your contract at the command line using `marlowe-cli`,* install [`marlowe-cli`](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-cli/ReadMe.md#installation) and follow the instructions [in the tutorial for running contracts with `marlowe-cli`](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-cli/lectures/04-marlowe-cli-concrete.md). Video lecture playlist [Marlowe CLI](https://www.youtube.com/playlist?list=PLNEK_Ejlx3x0GbvCw-61e9VfRafBT1JSw) provides an overview of `marlowe-cli`.
```

Reply:
Thanks! I'll commit a proposed revision shortly.

Reply:
Fixed in 7dd5c855fe4b8b02bf00f34d29c6dc9cb9e55da3.